rules/subsystem/subsystem.go

```diff
@@ -24,9 +25,16 @@ func init() {
 
 // ValidateSubsystem checks that the commit's subject is of the form '<subsystem>: <commit-msg>'.
 func ValidateSubsystem(c git.CommitEntry) (vr validate.Result) {
+	vr.CommitEntry = c
+	if len(strings.Split(c["parent"], " ")) > 1 {
+		vr.Pass = true
+		vr.Msg = "merge commits do not require subsystem in subject"
```

Comment:
Is this necessary? wouldn't we only expect merge commits to happen from the GH merge, which won't show up in the PR?